engines/sci/engine/script_patches.cpp

```diff
@@ -5647,38 +5647,38 @@ static const uint16 mothergoose256PatchReplay[] = {
 	0x39, 0x06,                      // pushi 06
 	0x76,                            // push0
 	0x76,                            // push0
-	0x38, PATCH_UINT16(200),         // pushi 200d
-	0x38, PATCH_UINT16(320),         // pushi 320d
+	0x38, PATCH_UINT16(0x00c8),      // pushi 200d
+	0x38, PATCH_UINT16(0x0140),      // pushi 320d
 	0x76,                            // push0
 	0x76,                            // push0
 	0x43, 0x15, 0x0c,                // callk SetPort -> set picture port to full screen
 	// 15 bytes
 	0x39, 0x04,                      // pushi 04
 	0x3c,                            // dup
 	0x76,                            // push0
-	0x38, PATCH_UINT16(255),         // pushi 255d
+	0x38, PATCH_UINT16(0x00ff),      // pushi 255d
 	0x76,                            // push0
 	0x43, 0x6f, 0x08,                // callk Palette -> set intensity to 0 for all colors
 	// 11 bytes
 	0x7a,                            // push2
-	0x38, PATCH_UINT16(800),         // pushi 800
+	0x38, PATCH_UINT16(0x0320),      // pushi 800
 	0x76,                            // push0
 	0x43, 0x08, 0x04,                // callk DrawPic -> draw picture 800
 	// 8 bytes
 	0x39, 0x06,                      // pushi 06
 	0x39, 0x0c,                      // pushi 0Ch
 	0x76,                            // push0
 	0x76,                            // push0
-	0x38, PATCH_UINT16(200),         // push 200
-	0x38, PATCH_UINT16(320),         // push 320
+	0x38, PATCH_UINT16(0x00c8),      // pushi 200
+	0x38, PATCH_UINT16(0x0140),      // pushi 320
 	0x78,                            // push1
```

Comment:
Same as before, I personally do not see a reason to change the decimal values here.
Those are meant to be 320x200 (screen size), that's why I used decimal notation instead of hexadecimal.